src/main/java/gregtechfoodoption/machines/multiblock/MetaTileEntitySteamBakingOven.java

```diff
@@ -0,0 +1,82 @@
+package gregtechfoodoption.machines.multiblock;
+
+import gregtech.api.capability.impl.SteamMultiWorkable;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.MetaTileEntityHolder;
+import gregtech.api.metatileentity.multiblock.IMultiblockPart;
+import gregtech.api.metatileentity.multiblock.RecipeMapSteamMultiblockController;
+import gregtech.api.pattern.BlockPattern;
+import gregtech.api.pattern.FactoryBlockPattern;
+import gregtech.api.recipes.RecipeMap;
+import gregtech.client.renderer.ICubeRenderer;
+import gregtech.common.blocks.BlockGlassCasing;
+import gregtech.common.blocks.MetaBlocks;
+import gregtechfoodoption.block.GTFOBlockCasing;
+import gregtechfoodoption.block.GTFOMetaBlocks;
+import gregtechfoodoption.client.GTFOClientHandler;
+import gregtechfoodoption.recipe.GTFORecipeMaps;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.ResourceLocation;
+
+import javax.annotation.Nonnull;
+
+import static gregtech.api.unification.material.Materials.Steel;
+
+public class MetaTileEntitySteamBakingOven extends RecipeMapSteamMultiblockController {
+    public MetaTileEntitySteamBakingOven(ResourceLocation metaTileEntityId, RecipeMap<?> recipeMap, double conversionRate) {
+        super(metaTileEntityId, GTFORecipeMaps.STEAM_BAKING_OVEN_RECIPE, CONVERSION_RATE);
```

Comment:
Although the recipe map does use temperature, this does not use it in the slightest.